cmd/root.go

```diff
@@ -0,0 +1,95 @@
+// Copyright © 2018 OrcaLLC
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
we should specify that this can be changed without notice if our pockets are light